src/Microsoft.ML.Data/Transforms/ValueMapping.cs

```diff
@@ -362,7 +362,7 @@ internal bool TryUnparse(StringBuilder sb)
             }
         }
 
-        public sealed class Arguments
+        internal sealed class Arguments
```

Comment:
>internal [](start = 8, length = 8)

For consistency, I would either make this public and only make specific field internal, or make the `Arguments` internal in the other transforms too.

Reply:
In other places we use `Arguments` classes as EntryPoint arguments. I don't want to get into business of cleaning entrypoints in this PR.
For this transform we don't have entrypoint, so I prefer to just hide it completely. 

---
In reply to: [252398379](https://github.com/dotnet/machinelearning/pull/2309#discussion_r252398379) [](ancestors = 252398379)

Reply:
Ok sounds good!

---
In reply to: [252399795](https://github.com/dotnet/machinelearning/pull/2309#discussion_r252399795) [](ancestors = 252399795,252398379)